content/releases/5/4_batch-delete/snippet.php

```diff
@@ -0,0 +1,33 @@
+<style>
+.v5-batch-columns {
+	display: grid;
+	gap: var(--spacing-6);
+	grid-template-areas:
+		"screenshot"
+		"teaser"
+		"code"
+}
+
+@media screen and (min-width: 80rem) {
+	.v5-batch-columns {
+		grid-template-columns: 1fr 1fr 2fr;
```

Comment:
Leaves the teaser and code boxes rather large with little content. Didn't find any good setup yesterday with the rather square image.

Reply:
I think this is fine. I've moved the code example to the right, which makes the layout a bit more interesting in my opinion.